pkg/sfs/share.go

```diff
@@ -19,27 +19,29 @@ package sfs
 import (
 	"github.com/huaweicloud/golangsdk"
 	"github.com/huaweicloud/golangsdk/openstack/sfs/v2/shares"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 	"k8s.io/klog"
 )
 
 const (
 	waitForAvailableShareTimeout = 3
 
-	shareAvailable     = "available"
+	shareAvailable = "available"
 
 	shareDescription = "provisioned-by=sfs.csi.huaweicloud.org"
 )
 
 func createShare(client *golangsdk.ServiceClient, createOpts *shares.CreateOpts) (*shares.Share, error) {
-    createOpts.Description = shareDescription
+	createOpts.Description = shareDescription
 	share, err := shares.Create(client, createOpts).Extract()
 	if err != nil {
-		return nil, err
+		return nil, status.Errorf(codes.Internal, "Failed to create share, %v", err)
 	}
 
 	err = waitForShareStatus(client, share.ID, shareAvailable, waitForAvailableShareTimeout)
 	if err != nil {
-		return nil, err
+		return nil, status.Errorf(codes.Internal, "Failed to wait for share create, %v", err)
```

Comment:
It is recommended to print `Error waiting for share to be created: %v`

Reply:
done